README.md

```diff
@@ -31,8 +31,9 @@ The following samples show different ways to use .NET Core images.
 ### Production
 
 - [dotnetapp-prod](dotnetapp-prod) - This sample is good for production since it relies on the .NET Core Runtime image, not the larger .NET Core SDK image. Most apps only need the runtime, reducing the size of your application image.
-- [dotnetapp-selfcontained](dotnetapp-selfcontained)- This sample is also good for production scenarios since it relies on an operating system image (without .NET Core). [Self-contained .NET Core apps](https://docs.microsoft.com/dotnet/articles/core/deploying/) include .NET Core as part of the app and not as a centrally installed component in a base image.
-- [dotnetapp-current](dotnetapp-current) -- This sample demonstrates how to configure an application to use the .NET Core 1.1 image. Both project.json and the Dockerfile have been updated to depend on .NET Core 1.1. This sample is the same as [dotnetapp-prod](dotnetapp-prod) with the exception of relying on a later .NET Core version.
+- [dotnetapp-selfcontained](dotnetapp-selfcontained) - This sample is also good for production scenarios since it relies on an operating system image (without .NET Core). [Self-contained .NET Core apps](https://docs.microsoft.com/dotnet/articles/core/deploying/) include .NET Core as part of the app and not as a centrally installed component in a base image.
+- [dotnetapp-current](dotnetapp-current) - This sample demonstrates how to configure an application to use the .NET Core 1.1 image. Both project.json and the Dockerfile have been updated to depend on .NET Core 1.1. This sample is the same as [dotnetapp-prod](dotnetapp-prod) with the exception of relying on a later .NET Core version.
```

Comment:
Looks like I forgot to update this - could you update the `project.json` reference to something like `the project file`